impl/src/templates/partials/rust_binary.template

```diff
@@ -36,4 +36,11 @@ rust_binary(
         "{{feature}}",
         {%- endfor %}
     ],
+    {%- if crate.aliased_dependencies | length != 0 %}
+    aliases = {
+        {%- for alias in crate.aliased_dependencies %}
+        {{alias}}
```

Comment:
Does this need to be fixed to match rust_library.template?

Reply:
Good catch, thank you